vnc_lite.html

```diff
@@ -179,7 +180,7 @@
 
         // Creating a new RFB object will start a new connection
         rfb = new RFB(document.getElementById('screen'), url,
-                      { credentials: { password: password } });
+                      { credentials: { password: password }, repeaterID: repeaterID });
```

Comment:
We want to keep `vnc_lite.html` minimal, so I'm afraid we don't want to add more features to it.

Reply:
Ok, I will change it in the next commit.